frontend/src/views/MovieDetailsScreen.vue

```diff
@@ -42,7 +42,7 @@ export default {
 
     async fetchMovie (id) {
       try {
-        const { data: movie } = await this.axios.get(`${import.meta.env.VITE_API_URL}/api/cinema/movies-${id}`, {
+        const { data: movie } = await this.axios.get(`${import.meta.env.VITE_API_URL}/api/cinema/movies-${id}/`, {
```

Comment:
Are you sure this endpoint exists on backend?
Also don't see screenshot of responses from developer tools